automation_tools/__init__.py

```diff
@@ -1741,9 +1741,11 @@ def apply_hotfix():
         run('/root/hotfix.sh')
 
 
-def upstream_install(admin_password=None, run_katello_installer=True,
-                     puppet4=True):
+def upstream_install(admin_password=None, run_katello_installer=True):
     """Task to install Foreman nightly using forklift scripts"""
+    sat_version = os.environ.get('SATELLITE_VERSION')
+    use_koji = 'koji' in sat_version
```

Comment:
SATELLITE_VERSION=koji -> install UPSTREAM out of koji (more recent but can be broken or fixed already, depends on nightly health)

Reply:
The values of SATELLITE_VERSION are fixed, it can only be the below:
```
6.1
6.2
6.3
6.4
upstream-nightly
downstream-nightly
```
What we do currently, for various INTERNAL builds is specify it under SATELLITE_DISTRIBUTION

```
INTERNAL
INTERNAL AK
INTERNAL REPOFILE
```
We would need to take similar approach here.

Reply:
I plan to add 'koji' in robottelo-ci later
this PR is not breaking anything without robottelo-ci changes

Reply:
I can get `6.3` using INTERNAL or INTERNAL AK and should have the same result only means to obtain the same version are different. 

While with koji and nightly also the result is different so not only the means to get that.
That why we have to treat them as a separate versions (so not distributions)